rustls/src/server/tls12.rs

```diff
@@ -191,6 +213,7 @@ mod client_hello {
                     )
                 })?;
 
+            // TODO (arash) needed for ffdhe?
```

Comment:
I think this is not needed when `self.suite.kx` is `DHE`. Because in this function, the ciphersuite is already decided.

So uou can refactor this by do `match` on `self.suite.kx`

Reply:
Thanks. Will update!